modules/core/src/main/java/org/apache/ignite/internal/util/distributed/DistributedProcess.java

```diff
@@ -367,6 +377,9 @@ private Process(UUID id) {
 
     /** Defines distributed processes. */
     public enum DistributedProcessType {
+        /** For test purpose only. */
```

Comment:
Personally, I don't support such cases. 
- we can use the `DEFAULT` type of the distributed process which can be used on all nodes. It's also nice to mention that a newly created distributed process must be registered on all nodes (e.g. via compute?).
- already existing types can be also reused in tests.

Reply:
I'm against using existing types. It'll require unnecessary configuration and logic for such processes and will make tests more difficult.

Reply:
What means the `DEFAULT` process? What if we want to add test logic to the action and the finish callbacks? When it will be registered?